src/terminal/parser/ut_parser/OutputEngineTest.cpp

```diff
@@ -1163,6 +1271,20 @@ class StatefulDispatch final : public TermDispatch
         return true;
     }
 
+    bool SetDefaultForeground(const DWORD color) noexcept override
+    {
+        _setDefaultForeground = true;
+        _defaultForegroundColor = color;
+        return true;
+    }
+
+    bool SetDefaultBackground(const DWORD color) noexcept override
+    {
+        _setDefaultBackground = true;
+        _defaultBackgroundColor = color;
+        return true;
+    }
+
     bool SetClipboard(std::wstring_view content) noexcept override
```

Comment:
So `SetColorTableEntry` is missing. I know. Right now I don't have enough knowledge to add tests cases about it. I think it's OK to leave it for the future.